apps/synapse-portal-framework/src/components/nf/NFBrowseToolsPage.tsx

```diff
@@ -162,7 +168,14 @@ const NFBrowseToolsPage = (props: NFBrowseToolsPageProps) => {
           <div className="searchToolsRow">
             <div className="searchInputWithIcon">
               <TextField
```

Comment:
I just realized that there's already a copy/paste of this search component in the ELBrowseToolsPage.  Could you refactor and reuse?  I know you asked about this, and at the time I didn't think it was needed, but you were right.